events.md

```diff
@@ -111,6 +112,17 @@ These JSON hashes included in the table above are broken out into their own tabl
 |location.location.latitude	|float	|A positive or negative float number representing the latitude of the address.
 |location.location.longitude	|float	|A positive or negative float number representing the longitude of the address.
 |location.location.accuracy	|enum	|A value representing the accuracy of the geocode. One of "Rooftop" or "Approximate".
+|location.public |boolean |Whether the venue's location should be shared publicly, or if false, only shared with RSVPs (for example, someone's house)
+
+#### Contact
+
+|Name          |Type      |Description
+|-----------    |-----------|--------------
+|contact.name	|string	|Name of the host or contact person for event (e.g., Jane Doe)
+|contact.email_address	|string	|Email address of the host (jane.doe@hotmail.co.uk)
+|contact.phone_number	|string	|Phone number of the host (214-555-0869)
+|contact.additional_info |string  |Free form place for information about the event contact

```

Comment:
Two small nitpicks:

The convention for descriptions seems to be to end with a period `.` and add any example value in parens after that.

```
A human readable identifier of the system where this event was created. (ex: “OSDI System”)
```

Additionally, we might consider adding `May contain text and/or HTML.` to the description for `contact.additional_info` (similar to the `instructions` field above).

Reply:
Do implementations out there contain HTML in that field? I know for us (and we're the ones who wanted that field) it's all plain text...